migrations/container/20201120090951_site-command_postfix_msmtrpc_fix.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace EE\Migration;
+
+use EE;
+use EE\RevertableStepProcessor;
+use EE\Model\Site;
+
+class PostfixMsmtrpcFix extends Base {
+
+	private $sites;
+	/** @var RevertableStepProcessor $rsp Keeps track of migration state. Reverts on error */
+	private static $rsp;
+
+	public function __construct() {
+
+		parent::__construct();
+		$this->sites = Site::all();
+		if ( $this->is_first_execution || ! $this->sites ) {
+			$this->skip_this_migration = true;
+		}
+	}
+
+	/**
+	 * Execute nginx config updates.
+	 *
+	 * @throws EE\ExitException
+	 */
+	public function up() {
+
+		if ( $this->skip_this_migration ) {
+			EE::debug( 'Skipping postfix-msmtrpc-fix migration as it is not needed.' );
+
+			return;
+		}
+
+		foreach ( $this->sites as $site ) {
+			mkdir( $site->site_fs_path . '/config/php/misc' );
```

Comment:
Check for site type here. If site type is HTML, then `continue` in loop.

Reply:
@mrrobot47 Done.